src/main/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
 <idea-plugin require-restart="true">
     <id>com.sourcegraph.jetbrains</id>
-    <name>Sourcegraph Cody + Code Search</name>
+    <name>Cody: AI Coding Assistant with Autocomplete & Chat</name>
```

Comment:
```
Execution failed for task ':patchPluginXml'.
> org.jdom2.input.JDOMParseException: Error on line 3: The entity name must immediately follow the '&' in the entity reference.
```

Unfortunately `&` will not work in xml file, it needs to be replaced with `&amp;`.
I will push fix if I can.